test.cpp

```
@@ -0,0 +1,48 @@
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+
+class Shape {
+public:
+    Shape() {}
```

Clang (14.0.6)

[error] constructor does not initialize these fields: size, color

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)